backend/game.py

```diff
@@ -42,79 +42,89 @@ def __init__(self, game_id: Text, options: GameOptions):
         self._game_id = game_id
 
     def play(self, tribe1: Tribe, tribe2: Tribe, gamedb: Database, engine: Engine) -> Player:
+        game_id = ""
```

Comment:
since _game_id is set in the Game constructor we can omit this check and just use self._game_id in the call to log_message.